docs/How-To_Guides/OpenStack/Barbican/generic-secret.md

```diff
@@ -0,0 +1,81 @@
+# Generic secret storage
+
+The simplest way to use Barbican is to create and retrieve a securely
+stored, generic secret.
+
+## How to store a generic secret
+
+It is possible to store any secret data with Barbican. The command
+below will create a secret of the type `passphrase`, named `mysecret`,
+which contains the passphrase `my very secret passphrase`.
+
+```bash
+openstack secret store \
+  --secret-type passphrase \
+  -p "my very secret passphrase" \
+  -n mysecret
+```
+```
++---------------+--------------------------------------------------------------------------------+
+| Field         | Value                                                                          |
++---------------+--------------------------------------------------------------------------------+
+| Secret href   | https://fra1.{{extra.brand_domain}}:9311/v1/secrets/33ef0985-f89e-4bf0-b318-887ecac0cba |
```

Comment:
In other places, you write `<region>`, would it make sense that we say, before the call, that we are using fra1 region for the example?

Reply:
In the console dumps I have preferred to be as close to the real thing as possible.

Reply:
I prefer it too.
I am just saying that because you don't write _fra1_ before, it might come as a surprise if the result is different.

But maybe I am too careful.

Reply:
The OpenStack CLI console dumps are a real issue, docs-wise, because they're incredibly wide. They're almost impossible to include even in the web output (without forcing the reader to scroll horizontally), and in portrait-mode PDF there isn't any chance at all to include them in a meaningful way. Other output formats besides the default `table` are hardly any better. `value` is useless (doesn't show names), `shell` and `json` are practically unreadable, `yaml` is... well, YAML.

Reply:
I agree with you on all your points.
I am just saying we didn't start by saying that "For this example, we'll use the Fra1 region".

That's all.

Reply:
Fixed in latest version.